tests/PHPStan/Analyser/data/integer-range-types.php

```diff
@@ -307,8 +307,8 @@ public function maximaInversion($rMin, $rMax) {
 		assertType('int<-5, max>', $rMin * -1);
 		assertType('int<min, -10>', $rMax * -2);
 
-		assertType('float|int<0, max>', -1 / $rMin);
-		assertType('float|int<min, 0>', -2 / $rMax);
+		assertType('-1|1|float', -1 / $rMin);
```

Comment:
-1 / $rMin with $rMin an int  will never be something else

Reply:
Wolframalpha is not doing great on this https://www.wolframalpha.com/input?i=%28-1%2Finterval%5B-9999%2C+5%5D+%29